docs/iot/guides/update-control-api.md

```diff
@@ -0,0 +1,134 @@
+# Tizen Update Control Native API
+
+- [Prerequisites](#prerequisites)
+- [Connecting to Server](#connecting-to-server)
+- [Checking the Latest Version](#checking-the-latest-version)
+- [Downloading and Updating to New Version](#downloading-and-updating-to-new-version)
+- [Making an Update Reservation](#making-an-update-reservation)
+- [Update Properties](#update-properties)
+
+You can control the system software version of IoT device using update control API.
+The control of system software version includes:
+- Checking the latest version
+- Downloading and updating to the latest version
+- Making a reservation for update
+
+The update procedure and the server might be different according to the platform developer.
+Therefore, this API just offers general functions for update, and platform developers must develop the application using update control API properly.
+
+>  **Note**
+>
+> As a default, the update agent works with [SmartThings Device Manager](https://console.smartthingsdm.com/) is available.
+
+## Prerequisites
+
+To enable your application to use the update control functionality:
+
+1. To use the update control API (in application), the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
```

Comment:
okay

Reply:
If so (only dedicated to IoT), please change the sentence as below.

```
To use the [update control](The expected api reference) API, the application..
```
@sabeena21, if only one profile is supported, please use this form.


Another question. Isn't this "update control" the name of the API?
If so, isn't "Update Control" correct?

Reply:
@mijin85cho Thanks for the comment. I will modify it soon.
However, I'm not sure which page has to be linked as "The expected api reference" because it seems that the API pages for update control API is not created yet like other IoT API (ex. Smart Things SDK (https://developer.tizen.org/dev-guide/things-sdk)).
Could you give me a guidance for this?

Reply:
@SunmLee if we don't have expected URL,
how about leaving it as a simple string for now (just "Update Control API")?